src/app/controllers/views/ControllerList/ControllerListTable/ControllerListTable.tsx

```diff
@@ -76,7 +96,58 @@ const generateRows = (
           "aria-label": "Type",
           className: "type-col",
           content: (
-            <span className="u-truncate">{controller.node_type_display}</span>
+            <span className="u-truncate">
+              {controller.node_type === NodeType.REGION_CONTROLLER ||
+              controller.node_type === NodeType.REGION_AND_RACK_CONTROLLER ? (
+                vaultEnabled ? (
+                  <Tooltip
+                    children={<Icon name="security-tick" />}
+                    message={
+                      <p>
+                        Vault is configured on this region controller for secret
+                        storage.
+                        <br />
+                        <a href={docsUrls.vaultIntegration}>
+                          Read more about Vault integration
+                        </a>
+                      </p>
+                    }
+                  />
+                ) : controller.vault_configured === true ? (
+                  unconfiguredControllers >= 1 && (
+                    <Tooltip
+                      children={<Icon name="security" />}
+                      message={
+                        <p>
+                          Vault is configured on this controller. <br />
+                          Once all controllers are configured, migrate the
+                          secrets. <br />
+                          <a href={docsUrls.vaultIntegration}>
+                            Read more about Vault integration
+                          </a>
+                        </p>
+                      }
+                    />
+                  )
+                ) : (
+                  configuredControllers >= 1 && (
+                    <Tooltip
+                      children={<Icon name="security-warning" />}
```

Comment:
At least the security warning icon should have a text description as it's not purely decorative.